README.md

```diff
@@ -90,10 +89,27 @@ end of the css file.
 
 Type: `Boolean`
 
-Render the banner all in one line
+Default: `false`
+
+Render the banner all in one line.
+
+NOTE: If the string contains new line chars, the resulting comment will
```

Comment:
To render the banner all in one line? I don't see anything wrong with that. As we already have that in the source code I suggest to leave that in place. I can think about real case scenarios where people wants to render the banner in multiline while on development mode but they want to render everything in one line on production